src/pages/learn/digital-gardening/index.tsx

```diff
@@ -122,12 +122,25 @@ export const digitalGardeningQuery = groq`*[_type == 'resource' && slug.current
   'cta': content[title == 'cta'][0]{
     description
   },
-  resources[]{
+  'featured': resources[title == 'Featured digital gardening courses'][0]{
+ 		'courses': resources[]{
+    	title,
+    	byline,
+      'name': content[title == 'name'][0].description,
+    	'path': resources[]->[0].path,
+    	'image': resources[]->[0].image
+  	}
+  },
```

Comment:
This displays weird on github but is formatted as you would expect.

resources is now a query for the resource titled 'Featured digital gardening courses'. Now we can add other sections to the page without the front end automatically rendering what it finds.

Reply:
This approach makes more sense!